Can_Node_Handler.cpp

```diff
@@ -83,8 +87,16 @@ void Can_Node_Handler::handleMessage(Frame& message) {
   const float throttleScaled = ((float)throttleExtended) * THROTTLE_SCALING_FACTOR;
   const int16_t outputTorque = (int16_t) (round(throttleScaled));
 
-  // Write torque commands
-  writeThrottleMessages(outputTorque);
+  if(analogThrottle>50){
+    // Write torque commands
+    writeThrottleMessages(outputTorque);
+  }
+  else if(wheel_RPM>2500){ 
+      writeThrottleMessages(-50);//Test Brake value when throttle unpressed
```

Comment:
The logic here is good, but it'll be pretty hard for us to see any regen behavior at -50: remember that the range of writeThrottleMessages is -2^15 to 2^15.